runtime/src/bank.rs

```diff
@@ -4058,22 +4055,6 @@ impl Bank {
             .verification_complete()
     }
 
-    pub fn get_fee_for_message_with_lamports_per_signature(
```

Comment:
fee_structure.calculate_fee() doesn't take `lamports_per_signature` anymore, this function can be fold into above `bank.get_fee_for_message()`.